roles/app/tasks/main.yml

```diff
@@ -134,12 +132,11 @@
       notify:
         - restart php72-php-fpm
 
-    - name: Remove known default distro files in /etc/opt/remi/php72/php-fpm.d/
-      file: path={{item}} state=absent
-      with_items:
-        - /etc/opt/remi/php72/php-fpm.d/www.conf
+    # Place empty www.conf to prevent this file being created on php-fpm OS-updates
+    - name: Add empty www.conf in /etc/opt/remi/php72/php-fpm.d/www.conf
+      copy: src='www.conf' dest='/etc/opt/remi/php72/php-fpm.d/www.conf'
       notify:
-        - restart php72-php-fpm
```

Comment:
This looks like a cut&paste error.
Must restart php72-php-fpm after changing it's config

Reply:
Thank you for pointing this out.